packages/components/card/src/asset-card/AssetCard.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { Flex } from '@contentful/f36-core';
+import { EntityStatusBadge } from '@contentful/f36-badge';
+import { Asset } from '@contentful/f36-asset';
+import type { AssetStatus, AssetType } from '@contentful/f36-asset';
+import { Icon } from '@contentful/f36-icon';
+
+import { BaseCard } from '../base-card/BaseCard';
+import type { BaseCardInternalProps } from '../base-card/BaseCard';
+import { CardActions } from '../base-card/CardActions';
+import { getAssetCardStyles } from './AssetCard.styles';
+
+const generateSizeStyles = ({
+  size,
+}: {
+  size: AssetCardInternalProps['size'];
+}): string => {
+  return css({
+    padding:
+      size === 'small'
+        ? tokens.spacingL
+        : `calc(1rem * (56 / ${tokens.fontBaseDefault}))`,
```

Comment:
Can we use spacing value here as well?

Reply:
This 56px is just for the Storybook preview, correct?